cpp/libcugraph_etl/src/renumbering.cu

```diff
@@ -16,9 +16,954 @@
+                           std::unique_ptr<cudf::column>(new cudf::column(
+                                                            cudf::data_type(cudf::type_id::INT32),
+                                                            0,
+                                                            rmm::device_buffer{0,cudaStream_t{0}})
+                                                          ),
+                            std::make_unique<cudf::table>(std::vector<std::unique_ptr<cudf::column>>{}));
+  }
+
+  template <typename Dtype,
+                std::enable_if_t<std::is_integral<Dtype>::value>* = nullptr>
+  std::tuple<std::unique_ptr<cudf::column>,
+        std::unique_ptr<cudf::column>,
+        std::unique_ptr<cudf::table>> operator() (cudf::table_view const& src_view,
+                                            cudf::table_view const& dst_view) {
+    assert(src_view.num_columns() == 2);
+    assert(dst_view.num_columns() == 2);
+
+    size_type num_rows = src_view.num_rows();
+    using char_type = int8_t;
+    using str_offset_type = int32_t;  // kernels init'd int32 only
+
+    std::vector<char_type*> src_vertex_chars_ptrs;
+    std::vector<str_offset_type*> src_vertex_offset_ptrs;
+    std::vector<char_type*> dst_vertex_chars_ptrs;
+    std::vector<str_offset_type*> dst_vertex_offset_ptrs;
+
+    for (int i = 0; i < src_view.num_columns(); i++) {
+      auto str_col_view = cudf::strings_column_view(src_view.column(i));
+      src_vertex_chars_ptrs.push_back(const_cast<char_type*>(str_col_view.chars().data<char_type>()));
+      src_vertex_offset_ptrs.push_back(const_cast<str_offset_type*>(
+                                            str_col_view.offsets().data<str_offset_type>()));
+    }
+
+    for (int i = 0; i < dst_view.num_columns(); i++) {
+      auto str_col_view = cudf::strings_column_view(dst_view.column(i));
+      dst_vertex_chars_ptrs.push_back(const_cast<char_type*>(str_col_view.chars().data<char_type>()));
+      dst_vertex_offset_ptrs.push_back(const_cast<str_offset_type*>(
+                                            str_col_view.offsets().data<str_offset_type>()));
+    }
+
+    accum_type *hist_insert_counter = nullptr;
+    CHECK_CUDA(cudaMallocHost(&hist_insert_counter, sizeof(accum_type)*32));
+    *hist_insert_counter = 0;
+
+    cudaStream_t exec_strm = NULL;
+    float load_factor = 0.7;
+
+    rmm::device_uvector<accum_type> atomic_agg(32, exec_strm);  // just padded to 32
+    CHECK_CUDA(cudaMemsetAsync(atomic_agg.data(), 0, sizeof(accum_type), exec_strm));
+
+    auto cuda_map_obj = cudf_map_type::create(std::max(
+                                                  static_cast<size_t>(
+                                                    static_cast<double>(num_rows) / load_factor),
+                                                    (size_t)num_rows + 1),
+                                              exec_strm,
+                                              str_hash_value{
```

Comment:
Your default constructor already uses these limits, you should just be able to use `str_hash_value{}` here.